pkg/build/controller/strategy/custom_test.go

```diff
@@ -57,20 +57,9 @@ func TestCustomCreateBuildPod(t *testing.T) {
 		t.Errorf("Expected never, got %#v", actual.Spec.RestartPolicy)
 	}
 
-	// expected volumes:
-	// docker socket
-	// push secret
-	// source secret
-	// additional secrets
-	// build-system-configmap
-	// certificate authorities
-	// container storage
-	// container run
-	// global CA injection configmap
```

Comment:
Why delete this comment instead of updating it?  The tests for the docker and source strategies will still include similar comments.

Reply:
Oops, I'll revert and update the comments accordingly.